.circleci/config.yml

```diff
@@ -0,0 +1,134 @@
+### AUTO GENERATED. DO NOT MODIFY. ###
```

Comment:
There is never need to review this file. This is autogenerated/put together with all files in .circleci/src with the script `./tools/circleci-update-config`. You need to install the CLI tool for this to work.